__tests__/app.test.js

```diff
@@ -296,3 +296,22 @@ describe("POST: /api/articles/:article_id/comments", () => {
         .expect(404)
     });
 })
+
+describe("DELETE /api/comments/:comment_id", () => {
+    test("200: Successfully deletes comment and returns deleted comment", () => {
+        return request(app)
+        .delete('/api/comments/1')
+        .expect(200)
+        .then((res) => {
+            expect(res.body.comment).toMatchObject({
+                comment_id: 1,
+                body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+                article_id: 9,
+                author: "butter_bridge",
+                votes: 16,
+                created_at: "2020-04-06T12:17:00.000Z"
+            })
+        })
+        
+    })
+})
```

Comment:
There can and should be error handling done for this endpoint as well. 

What should happen when a user gives an article id which is valid but does not exist?
What should happen when a user gives an invalid article id?